### ISSUE TITLE
[DOC]

### ISSUE THREAD

**Issue by prepstarr** [django-cms/django-cms#7614]
## Description
The documentation does not explain how to get it into the admin panel ... they just show so if someone was following the steps then they would not be able to arrive at the solution... unfortunately causing me to be stuck ... I just wanted this in the admin panel and that's it.. done...

## Do you want to help fix this documentation issue?

* [ X] Yes, I want to help fix this issue and I will join #workgroup-documentation on [Slack](https://www.django-cms.org/slack) to confirm with the team that a PR is welcome. 
* [ ] No, I only want to report the issue.

**Comment by prepstarr** [comment]
I got it into the admin panel it needed the media folder created but now I followed the directions and there's an issue with the downloading and uploading of files...

**Comment by prepstarr** [comment]
https://github.com/django-cms/django-filer/issues/1401

**Comment by prepstarr** [comment]
it feels like there's some things missing in the documentation ...

**Comment by prepstarr** [comment]
you must add a /media folder in your project directory

**Comment by fsbraun** [comment]
Does this refer to filer?

**Comment by prepstarr** [comment]
> 

yes this refers to filer setup if the environment does not contain media_root and a media directory than filer does not work ...

also for apache setup cause it's working now thanks to contributors on the Slack channel please add the following I am sending it the credit of the person but you could omit that if you wish to or cite it... basically the proxy pass part is not common sense , I mean if you think about it yes but then it didn't work with django on its own so I deduced it was not fully an apache issue but it was cause it requires the full setup perhaps... I don't know the program is working now but these addons to the instructions would help someone else like myself ...

Corentin Bettiol
  [3:39 AM](https://django-cmsworkspace.slack.com/archives/C01EBCEK49E/p1690789175344569)
[@Paul Kagan](https://django-cmsworkspace.slack.com/team/U05KSV181JM)
 You might want to add those lines in your apache config :
    ProxyPass /static/ !
    ProxyPass /media/ !
